test/runtime/samples/store-contextual/_config.js

```diff
@@ -0,0 +1,76 @@
+import { writable } from '../../../../store.js';
+
+const todos = [
+	writable({ done: false, text: 'write docs' }),
+	writable({ done: false, text: 'implement contextual stores' }),
+	writable({ done: false, text: 'go outside' })
+];
+
+export default {
+	error: `Stores must be declared at the top level of the component (this may change in a future version of Svelte)`,
```

Comment:
I assume the other values in this object don't do anything when `error` is present? So we can just remove `error` once this is implemented?

Reply:
exactly, yep